content/repositories.md

```diff
@@ -0,0 +1,20 @@
+
+More and more, repositories are becoming critical in [Data Management Plans (DMP)](data-management-plan.md) when applying for funding or setting up research projects. Not only do repositories help archive information over time, but they also allow you to increase the value of your work by allowing it to be incorporated into other work. Additionally, by documenting and storing your information, you can save time down the road when aspects of a project raise questions (at some point we all spend half a day peering over old notes to find the answer to a small question). 
+
+Repositories are different from the typical local or cloud storage platforms: These storage platforms do not certify as an archive and could disappear at any time (think of Dropbox, GitHub, etc.). Hard drives can fail, online backups can disappear. Repositories adapt a longer timeframe and therefore provide an archival service more than anything else. However, within repositories there are variations too: [EASY-DANS](http://easy.dans.knaw.nl/) archives data with the goal of persistence over the centuries, whereas [Figshare](https://figshare.com) or the [Open Science Framework](https://osf.io) archive with the goal of persistence over the decades.
+
+Another aspect of archival means that when other people want to build on top of your information in their research, they need to be assured that they can refer back to what they used in a research paper. This is why many repositories provide you with a [Digital Object Identifier (DOI)](digital-object-identifier.md), providing a stable link to your information (see also [Reference rot](reference-rot.md)).
+
+Many repositories have unique features that can help you out in your research and increasing the reach of the work you do. There is no one answer to store your information, and every person might have a different workflow that fits a different repository. For example, if you use [GitHub](https://github.com) for your [Project Management](project-management.md), you can also log in on [Zenodo](https://zenodo.org) and archive your project with a few simple steps.
+
+
+# Exercises
+
+* What kind of information could you archive within a research project? Think about what information you produce, what information might be valuable for other research projects.
+* Use the [re3data.org](https://www.re3data.org/browse/by-country/) country browser to find a repository to archive your data at the end of a project. There are many various features, so check out a few or ask people you know!
```

Comment:
For the second exercise (or can be added as a fourth exercise), you could include for them to also check out the repositories their University/Institute recommends.

e.g. could change to: "...There are many various features, so check out a few, see what your University/Institute recommends or ask people you know!"

(In my case, the [University of Manchester Library provides](http://www.library.manchester.ac.uk/using-the-library/staff/research/services/research-data-management/sharing/): "If no discipline-specific repository is available then Mendeley Data is the University of Manchester’s recommended general-purpose, open research data repository. Details of data deposited in Mendeley Data will automatically be added to your Pure account and your researcher profile. Other general-purpose data repositories are also available, e.g. Zenodo and Figshare.")

Reply:
Good suggestion! I am trying to refrain from doing to many institute specific references because some people are not at organizations. But it makes sense here. Going to update now. 🚶